go/store/nbs/manifest.go

```diff
@@ -501,7 +501,7 @@ func formatSpecs(specs []tableSpec, tableInfo []string) {
 // persisted manifest against the lock hash it saw last time it loaded the
 // contents of a manifest. If they do not match, the client must not update
 // the persisted manifest.
-func generateLockHash(root hash.Hash, specs []tableSpec, appendix []tableSpec) hash.Hash {
+func generateLockHash(root hash.Hash, specs []tableSpec, appendix []tableSpec, extra []byte) hash.Hash {
```

Comment:
I'm a bit skeptical about this. Using arbitrary extra bytes makes it extensible but feels messy. I'm going to keep reading and circle back on this.

Reply:
I think this is fine for now. If we ever need to change the manifest format again we can think about it more.

Reply:
Yeah, agreed this is not the greatest, but it works for now.